src/vmm/src/resources.rs

```diff
@@ -1368,6 +1385,21 @@ mod tests {
         assert!(vm_resources.set_balloon_device(new_balloon_cfg).is_err());
     }
 
+    #[test]
+    fn test_set_entropy_device() {
+        let mut vm_resources = default_vm_resources();
+        vm_resources.entropy = EntropyDeviceBuilder::new();
+        let entropy_device_cfg = EntropyDeviceConfig {};
+
+        assert!(vm_resources.entropy.get().is_none());
```

Comment:
Nit: I tend to prefer using `assert_eq` to be more inline with the other specific asserts, but this is very picky.
```suggestion
        assert_eq!(vm_resources.entropy.get(),None);
```

Reply:
Can't do, because `EntropyDeviceBuilder` does not implement `Debug`. I tried to add it but then I need to derive `Debug` for `Entropy` which includes other objects that do not implement it and I'm not touching in this PR, so I'd like to avoid doing it.

Reply:
... in this PR

Reply:
We could avoid this in the future by using [`#![warn(missing_debug_implementations)]`](https://doc.rust-lang.org/stable/nightly-rustc/rustc_lint/builtin/static.MISSING_DEBUG_IMPLEMENTATIONS.html), maybe it is worth creating an issue for implementing this lint?

Reply:
Not sure we want to commit to create debug implementations for all of our types.

Reply:
I think we should commit to fixing this where we encounter it, or we need to commit such that don't encounter it. Otherwise I think every PR which touches these types will have the same reasoning that they don't think fixing them fits in their PR so will not test as thoroughly.